tools/internal_ci/macos/grpc_run_bazel_cpp_ios_tests.sh

```diff
@@ -21,14 +21,25 @@ source $(dirname $0)/../../../tools/internal_ci/helper_scripts/move_src_tree_and
 # change to grpc repo root
 cd $(dirname $0)/../../..
 
+source tools/internal_ci/helper_scripts/prepare_build_macos_rc
+
+# make sure bazel is available
+tools/bazel version
+
 ./tools/run_tests/start_port_server.py
 
-dirs=(end2end server client common codegen util grpclb test)
-for dir in ${dirs[*]}; do
-  echo $dir
-  out=`tools/bazel query "kind(ios_unit_test, tests(//test/cpp/$dir/...))" 2>/dev/null | grep '^//'`
-  for test in $out; do
-    echo "Running: $test"
-    tools/bazel test --test_summary=detailed --test_output=all $test
-  done
-done
+# only select ObjC test from the following subdirs
+# TODO(jtattermusch): start running selected tests from //test/core too.
+test_pattern="//test/cpp/end2end/... + //test/cpp/server/... + //test/cpp/client/... + //test/cpp/common/... + //test/cpp/codegen/... + //test/cpp/util/... + //test/cpp/grpclb/... + //test/cpp/test/..."
```

Comment:
nit: can iOS share & reuse a test pattern list from cpp or other lang ? or is this need to be iOS specific ?